pkg/config/process.go

```diff
@@ -0,0 +1,50 @@
+package config
+
+import (
+	"os"
+	"time"
+)
+
+func setupProcesses(config Config) {
+	config.SetDefault("process_config.enabled", "false")
+	// process_config.enabled is only used on Windows by the core agent to start the process agent service.
+	// it can be set from file, but not from env. Override it with value from DD_PROCESS_AGENT_ENABLED.
+	ddProcessAgentEnabled, found := os.LookupEnv("DD_PROCESS_AGENT_ENABLED")
```

Comment:
I thought the plan was to configure DD_PROCESS_CONFIG_ and DD_PROCESS_AGENT_ prefixes universally for every setting?

Reply:
Yep, this part has yet to be changed though. I just moved this part to the process file so we can group all of the process settings.